common/scripts/.eslintrc.ts.json

```diff
@@ -8,6 +8,6 @@
     "@typescript-eslint/prefer-optional-chain": "warn",
     "@typescript-eslint/prefer-nullish-coalescing": "warn",
     "react-hooks/exhaustive-deps": "warn",
-    "deprecation/deprecation": "error"
+    "deprecation/deprecation": "warn"
```

Comment:
lets revert this

Reply:
Figured this out.. for some reason cannot ignore the deprecation errors if I leave the deprecation/deprecation rule here. If I move it out to the base eslint config, then it works 🤷‍♂️

Reply:
might be due to multiple version of typescript-eslint in the repo now due to using cra5 for the test-apps which pulls in ts-eslint@5, and eslint@8. i might try to fix this in a different pr